index.d.ts

```diff
@@ -0,0 +1,33 @@
+export interface Options {
+	/**
+	 * Overwrite existing destination file.
+	 *
+	 * @default true
+	 */
+	readonly overwrite?: boolean;
+}
+
+/**
+ * Move a file.
+ *
+ * @param source - File you want to move.
+ * @param destination - Where you want the file moved.
+ * @returns A `Promise`.
```

Comment:
I just realized that this is not a useful description. I know you took it from the readme, but feel free to improve anything or comment if something could be improved. I really want the docs and everything to be very high quality.